skbio/alignment/_tabular_msa.py

```diff
@@ -201,6 +201,16 @@ def keys(self):
 
     @property
     @experimental(as_of='0.4.0-dev')
+    def minter(self):
+        if self._minter is not None:
+            return self._minter
+        else:
+            raise OperationError(
+                "Attempted use of key minter on MSA, but none exists. Use "
```

Comment:
Minor: the user isn't really *using* the minter on the MSA, just requesting it. Maybe reword to something like "Minter does not exist. Use `reindex` with the `minter` parameter to set one."